perun/logic/config.py

```diff
@@ -173,7 +173,6 @@ def write_config_to(path: str, config_data: dict[str, Any]) -> None:
     :param str path: path where the config will be stored to
     :param dict config_data: dictionary with contents of the configuration
     """
-    perun_log.msg_to_stdout("Writing config '{}' at {}".format(config_data, path), 2)
```

Comment:
There seems to be no replacement for this message. Is this intended or just a mistake?

Reply:
Yes, it was intended.